redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UICollectionViewListUpdateCallback.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2023 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.lazylayout.uiview
+
+import app.cash.redwood.lazylayout.widget.ListUpdateCallback
+import platform.UIKit.UICollectionView
+
+internal class UICollectionViewListUpdateCallback(
```

Comment:
This is currently a no-op. The atomic change events for `UICollectionView` will be done in a follow-up.